Dockerfile

```diff
@@ -3,20 +3,23 @@ FROM docker.io/centos:centos7
 RUN yum install -y python-requests && \
     curl https://raw.githubusercontent.com/openstack/tripleo-repos/5609d2e3aee35578e914bcbfac92a46c759c0a31/tripleo_repos/main.py | python - current && \
     yum install -y openstack-ironic-api openstack-ironic-conductor crudini iproute dnsmasq httpd qemu-img-ev iscsi-initiator-utils parted gdisk ipxe-bootimgs psmisc sysvinit-tools mariadb-server python-PyMySQL python2-chardet && \
+    yum install -y epel-release python-pip python-devel gcc && \
```

Comment:
maybe we can add this to the yum command on line 5 instead

Reply:
Done

Reply:
Can we also leave a TODO to remove epel-release? We cannot use it in the final product.

Reply:
@dtantsur something like: `# TODO(iurygregory): remove epel-release and  other packages necessary for the ironic_prometheus_exporter when we have a package for it` ?